src/reynir_correct/settings.py

```diff
@@ -478,9 +488,160 @@ def add(morph: str, boundlist: List[str], freelist: List[str]) -> None:
+        else:
+            return ""
+
+    @staticmethod
+    def get_tag(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][4]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][4]
+        else:
+            return ""
+
+    @staticmethod
+    def get_eink(wrong_form: str) -> int:
+        d = Ritmyndir.DICT
+        eink = 1
+        if wrong_form in d:
+            eink = Ritmyndir.DICT[wrong_form][5]
+        elif wrong_form.lower() in d:
+            eink = Ritmyndir.DICT[wrong_form.lower()][5]
+        return eink
+
+    @staticmethod
+    def get_malsnid(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][6].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][6].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_stafs(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][7].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][7].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_aslatt(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][8].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][8].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_beyg(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][9].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][9].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_code(wrong_form: str) -> str:
+        code = ""
+        w = wrong_form
+        if wrong_form not in Ritmyndir.DICT:
+            if wrong_form.lower() in Ritmyndir.DICT:
+                w = wrong_form.lower()
+            else:
+                return "R001"
+        code = Ritmyndir.get_stafs(wrong_form)
+        if not code:
+            code = Ritmyndir.get_aslatt(wrong_form)
+        if not code:
+            code = Ritmyndir.get_beyg(wrong_form)
+        if not code:
+            code = Ritmyndir.get_malsnid(wrong_form)
+        if not code:
+            code = R_EINKUNN[Ritmyndir.get_eink(wrong_form)]
+        return code
+
+    @staticmethod
+    def add(wrong_form: str, details: RitmyndirTuple) -> None:
+        # TODO Same ritmynd can occur multiple times in the data from different references, how to handle?
+        # TODO Also check if the same ritmynd has many different corrections in the data,
```

Comment:
Could potentially make the DICT entries a list of corrections, instead of a single correction only. A defaultdict() would then be a good choice instead of a plain dict.

Reply:
A plausible idea; added a comment for later implementation.